Assignment-03/assignment 03-1.3 .c

```diff
@@ -0,0 +1,13 @@
+#include<stdio.h>
+//author sidhant
+//date of creation
+int main()
+{
+    char str1[10],str2[10];
+    printf("enter the first name \n");    //input the first name
+    scanf("%s",&str1);

```

Comment:
If you know the length of the string , you should consider using [scanf_s()](https://linux.die.net/man/3/scanf) instead:

```
// Do not forget to account for the trailing null byte
scanf_s("%s", &str1, 9);
str1[9] = '\0';
```